packages/core/src/js/tracing/span.ts

```diff
@@ -154,3 +154,30 @@ export function addDefaultOpForSpanFrom(client: Client): void {
     }
   });
 }
+
+export const SPAN_THREAD_ID = 'thread.id';
+export const SPAN_THREAD_ID_MAIN = 1;
+export const SPAN_THREAD_ID_JAVASCRIPT = 2;
+export const SPAN_THREAD_NAME = 'thread.name';
+export const SPAN_THREAD_NAME_MAIN = 'main';
+export const SPAN_THREAD_NAME_JAVASCRIPT = 'javascript';
+
+const UI_THREAD_PREFIX = 'ui.';
```

Comment:
The `ui.` prefix doesn't indicate that the span is from the main thread. Currently all the spans created in RN are from the JS thread.

The only exception are app start spans which are fetched from the native SDKs. https://github.com/getsentry/sentry-react-native/blob/18ce5e87b6e981e23ef9493207fb152d9c6aa8e7/packages/core/src/js/tracing/integrations/appStart.ts#L310

These should be marked as main thread.

Reply:
Thank you for pointing me to the right direction @krystofwoldrich 🙇 
Updated with 377fa6a5bfda9f0b52df139d6e2bbf28441328f0